src/main/java/io/netty/incubator/channel/uring/IOUring.java

```diff
@@ -28,6 +28,9 @@
         if (SystemPropertyUtil.getBoolean("io.netty.transport.noNative", false)) {
             cause = new UnsupportedOperationException(
                     "Native transport was explicit disabled with -Dio.netty.transport.noNative=true");
+        } else if (!Native.checkKernelVersion(Native.kernelVersion())) {
+            cause = new UnsupportedOperationException(
+                    "you need at least kernel version 5.9");
```

Comment:
nit: let's include what Native.kernelVersion() returned in the exception message

Reply:
actually good idea :+1: